src/components/items-payment/index.scss

```diff
@@ -67,27 +61,21 @@
   display: flex;
   flex-direction: column;
 }
-
 /* Estilos para los campos de selección */
```

Comment:
Tech comments in english (if they are necessary), please change the language, thx.

Reply:
This component was made by Rodrigo, I have not made any changes to it